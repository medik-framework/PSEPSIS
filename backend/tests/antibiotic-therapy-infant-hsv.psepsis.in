[
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AgeEntered",
    "eventArgs": [28]
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "WeightEntered",
    "eventArgs": [6],
    "waitFor": {
      "name": "ConfirmAgeEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "StartAntibioticTherapy",
    "eventArgs": [],
    "waitFor": {
      "name": "ConfirmWeightEntered",
      "args": []
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "AntibioticsPrimaryConditionsEntered",
    "eventArgs": [false, false, false, false, false, false],
    "waitFor": {
      "name": "Instruct",
      "args": ["get antibiotics primary conditions"]
    }
  },
  {
    "id": "tabletApp",
    "action": "broadcast",
    "eventName": "HSVConcernEntered",
    "eventArgs": [],
    "waitFor": {
      "name": "RecommendDrug"
    }
  },
  {
    "action":"exit",
    "waitFor": {
      "name": "Recommend"
    }
  }
]
