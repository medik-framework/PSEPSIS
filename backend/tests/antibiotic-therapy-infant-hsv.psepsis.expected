[
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmAgeEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "ConfirmWeightEntered",
    "args": []
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Instruct",
    "args": ["get antibiotics primary conditions"]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "RecommendDrug",
    "args": [ "Infants 0 to 28 days of age",
              "OSF Antibiotics Set",
              "drug combination",
              "ampicillin (50 mg/kg for the initial dose)",
              "cefotaxime (50 mg/kg for the initial dose)",
              "gentamycin (2.5 mg/kg for the initial dose)"
    ]
  },
  {
    "id": "tabletApp",
    "interface": "TabletApp",
    "name": "Recommend",
    "args": [ "consider additional antibiotics",
              "ADD acyclovir (20 mg/kg per dose) for suspicion of HSV infection"
    ]
  }
]

